components/Avatar/avatar.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Can add meta descriptions to personalise it more.
Like <meta name="author" content="Viresh" />
    <meta name="description" content="about your ui" />